apps/deploy-web/src/components/user/UserSettingsForm.tsx

```diff
@@ -83,98 +83,100 @@ export const UserSettingsForm: React.FunctionComponent = () => {
   return (
     <Layout isLoading={isLoading}>
       <NextSeo title={user?.username} />
-      <UserProfileLayout page="settings" username={user?.username} bio={user?.bio}>
-        {isLoading || !user ? (
-          <div className="flex items-center justify-center p-8">
-            <Spinner size="large" />
-          </div>
-        ) : (
-          <FormPaper>
-            <form onSubmit={handleSubmit(onSubmit)}>
-              <LabelValue label="Email" value={user.email} />
-              <LabelValue
-                label="Username"
-                value={
-                  <>
+      {user?.username && user?.bio && (
+        <UserProfileLayout page="settings" username={user?.username} bio={user?.bio}>
+          {isLoading || !user ? (
```

Comment:
I guess we don't need to check user if the user has a username.
```suggestion
          {isLoading ? (
```

Reply:
ok, to handle cases like this added a container which would provide user guaranteed. 
https://github.com/akash-network/console/pull/265/commits/5487667ac93d0acc80c1fcc57113b52390ebeb27